types/stripe/index.d.ts

```diff
@@ -4642,9 +4642,10 @@ declare namespace Stripe {
             items: IList<subscriptionItems.ISubscriptionItem>;
 
             /**
-             * Hash describing the plan the customer is subscribed to
+             * Hash describing the plan the customer is subscribed to.  Only set if the subscription
+             * contains a single plan.
              */
-            plan: plans.IPlan;
+            plan?: plans.IPlan | null;
```

Comment:
Just wanted to double check on the intent here. Either the property is not set (undefined), or it is set with a valid value. Is it intentional that the value can be set to be `null`? From the docs, it seems like the if it is set, it will be a single plan (which I would assume is not going to be `null`).

Reply:
Currently the API does return `null`.  However, the docs say it isn't set, so I guess I'm leaving it open to both here.

![image](https://user-images.githubusercontent.com/327833/51873712-1b3f6100-2313-11e9-9567-0733c0d44ea5.png)

Reply:
the nullable types are a bit of a mess at the moment, the Stripe documentation isn't the best unfortunately 😞 I got a response from an employee of Stripe to use their OpenAPI spec as the "golden standard"

https://github.com/DefinitelyTyped/DefinitelyTyped/issues/27656